packages/server/api/.env

```diff
@@ -15,4 +15,4 @@ AP_TELEMETRY_ENABLED=false
 AP_TRIGGER_DEFAULT_POLL_INTERVAL=1
 AP_WEBHOOK_TIMEOUT_SECONDS=30
 AP_PIECES_SYNC_MODE=NONE
-AP_WEBHOOK_URL="http://localhost:3000"
```

Comment:
Can you please reverse the whitespace there, We have hooks to forbid commit .env and this will cause an issue for other when they are merging.

Reply:
done 👍 , thnx